requirements-test.txt

```diff
@@ -1,6 +1,14 @@
 -r requirements.txt
 
+tox
+
 flake8<3.0
+
+pylint==1.6.4 ; python_version > '2.6'
+pylint==1.3.1 ; python_version <= '2.6'
+astroid==1.3.5 ; python_version <= '2.6'
```

Comment:
What's this?

Reply:
This was to get Python 2.6 builds to work , as pylint dropped support for Python 2.6 a while ago, hence this workaround.
Anyway, https://github.com/html5lib/html5lib-python/issues/330 has landed, so that is unnecessary now.